public/js/search.js

```diff
@@ -70,34 +74,22 @@ function displayMovieInfo() {
             
             renderButtons()
             saveMovies()
-        }).then((response) => {
-          if (response.length = 0)
-          fetch('/api/movies', {
-            method: 'POST',
-            headers: {
-              'Content-Type': 'application/json',
-            },
-            body: JSON.stringify(review),
-          })
         })
         .catch((error) => {
           console.error('Error:', error);
         });
-    };
-
-
 
 //On click event listener for search button
 $("#run-search").on("click", function () {
-    movie = $("#search-term").val()
-    displayMovieInfo()
+    const movie = $("#search-term").val()
+    displayMovieInfo(movie)
 
 })
 
 //On click event listener for movie buttons
 $(document).on("click", ".movie-btn", function () {
-    movie = $(this).attr("data-Title");
-    displayMovieInfo()
+    const movie = $(this).attr("data-Title");
+    displayMovieInfo() // TODO: should navigate to the reviews page with the selected movie
```

Comment:
This needs a parameter

Reply:
`$(document).on("click", ".movie-btn", function myFunction() {
    location.replace("/addreview/" + movie.title);
});`

Reply:
Confirm that `movie.title` is the correct property